tools/Azure.Mcp.Tools.Communication/src/Commands/Sms/SmsSendCommand.cs

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.CommandLine;
+using System.CommandLine.Parsing;
+using System.Net;
+using Azure.Mcp.Core.Commands;
+using Azure.Mcp.Core.Extensions;
+using Azure.Mcp.Core.Models.Command;
+using Azure.Mcp.Core.Models.Option;
+using Azure.Mcp.Tools.Communication.Models;
+using Azure.Mcp.Tools.Communication.Options;
+using Azure.Mcp.Tools.Communication.Options.Sms;
+using Azure.Mcp.Tools.Communication.Services;
+using Microsoft.Extensions.Logging;
+
+namespace Azure.Mcp.Tools.Communication.Commands.Sms;
+
+public sealed class SmsSendCommand(ILogger<SmsSendCommand> logger) : BaseCommunicationCommand<SmsSendOptions>
+{
+    private const string CommandTitle = "Send SMS Message";
+    private readonly ILogger<SmsSendCommand> _logger = logger;
+
+    // Define options from OptionDefinitions
+    private readonly Option<string> _fromOption = CommunicationOptionDefinitions.From;
+    private readonly Option<string[]> _toOption = CommunicationOptionDefinitions.To;
+    private readonly Option<string> _messageOption = CommunicationOptionDefinitions.Message;
+    private readonly Option<bool> _enableDeliveryReportOption = CommunicationOptionDefinitions.EnableDeliveryReport;
+    private readonly Option<string> _tagOption = CommunicationOptionDefinitions.Tag;
+
+    public override string Name => "send";
+
+    public override string Description =>
+        """
+        Send an SMS message using Azure Communication Services.
+        
+        Sends SMS messages to one or more recipients using your Communication Services resource.
+        Supports delivery reporting and custom tags for message tracking.
+        
+        Required options:
+        - --connection-string: Azure Communication Services connection string
+        - --from: SMS-enabled phone number (E.164 format, e.g., +14255550123)
+        - --to: Recipient phone number(s) (E.164 format)
+        - --message: SMS message content
+        
+        Returns: Array of SMS send results with message IDs and delivery status.
+        """;
+
+    public override string Title => CommandTitle;
+
+    public override ToolMetadata Metadata => new()
+    {
+        Destructive = false,
```

Comment:
Needs to be secret as well. @xiangyan99

Reply:
@KarishmaGhiya the `secret` property refers to any sensitive data, not only secrets. Due to the nature of this tool, the message contents could be considered confidential.

https://github.com/microsoft/mcp/blob/413d9aba7622f2747d0cfb42e171db657fccbcaa/core/Azure.Mcp.Core/src/Commands/ToolMetadata.cs#L78-L79

Reply:
Ok thanks for confirming, I thought it was just the return type. I changed the secret to true.